### PR TITLE
Remove adapter options. Update QC to allow adapter adaptation.

### PR DESCRIPTION
**Pull request by rhpvorderman**

### CODE REVIEW

File: readgroup.wdl [hunk 1]
```diff
@@ -66,39 +64,34 @@ workflow Readgroup {
...
+    if (numberChunks > 1) {
+        call biopet.FastqSplitter as fastqsplitterR1 {
             input:
-                inputFastq = select_first([copyR2.outputFile]),
-                outputPaths = chunksR2,
+                inputFastq = copyR1.outputFile,
+                outputPaths = chunksR1,
                 dockerTag = dockerTags["biopet-fastqsplitter"]
         }
+        if (defined(readgroup.reads.R2)){
```
Comment (DavyCats) [inline readgroup.wdl]: Do `if` statements work inside of `if` statements nowadays?

File: readgroup.wdl [hunk 2]
```diff
@@ -66,39 +64,34 @@ workflow Readgroup {
...
-            then {"R1": fastqsplitterR1.chunks[x],
-                "R2": select_first([fastqsplitterR2.chunks])[x]}
-            else {"R1": fastqsplitterR1.chunks[x]}
-    }
 
     # QC and Mapping
-    scatter (chunk in chunks) {
-        String chunkDir = sub(chunk.R1, basename(chunk.R1), "")
+    scatter (x in range(length(chunksR1))) {
+        File  chunk_read1 = if defined(fastqsplitterR1.chunks) then select_first([fastqsplitterR1.chunks])[x] else reads.R1
```
Comment (DavyCats) [inline readgroup.wdl]: Couldn't `select_all` be used here, instead of putting the array in an array first?

File: readgroup.wdl (hunk 1 above)
Comment (rhpvorderman) [reply readgroup.wdl]: Yes. They always have worked.

File: readgroup.wdl (hunk 2 above)
Comment (rhpvorderman) [reply readgroup.wdl]: No, because it is the array itself which is optional. Not the elements in it.

**Pull request merged**